src/index.spec.js

```diff
@@ -580,15 +580,17 @@ describe('Dropzone', () => {
         dataTransfer: { files: images.concat(files) }
       })
       const rejected = dropSpy.firstCall.args[1]
-      expect(rejected.length).toEqual(2)
+      expect(rejected.length).toBeGreaterThan(1)
     })
 
     it('should allow single files to be dropped if multiple is false', () => {
```

Comment:
The test name is misleading

Reply:
changing the test name according to its behaviour